server/src/db/migrations/20210223211631-create-infrastructure-types.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('infrastructureTypes', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.STRING,
+        defaultValue: Sequelize.literal('uuid_generate_v4()'),
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('infrastructureTypes');
+  },
+};
```

Comment:
What's with the filename? Is this how they're wanting it labeled out?

Reply:
yeah - the table name is a part of the ticket